2020-10-21 18:43:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "IOException reading Item_table_1.xlsx"; SQL statement:
CREATE TABLE ITEM AS SELECT * FROM CSVREAD('Item_table_1.xlsx') [90028-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.tools.Csv.convertException(Csv.java:586)
	at org.h2.tools.Csv.read(Csv.java:194)
	at org.h2.expression.function.Function.getValueForColumnList(Function.java:2728)
	at org.h2.table.FunctionTable.<init>(FunctionTable.java:60)
	at org.h2.command.Parser.readTableFunction(Parser.java:1945)
	at org.h2.command.Parser.readTableFilter(Parser.java:1892)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:2641)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:2788)
	at org.h2.command.Parser.parseSelectSub(Parser.java:2636)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:2469)
	at org.h2.command.Parser.parseSelect(Parser.java:2440)
	at org.h2.command.Parser.parseCreateTable(Parser.java:7906)
	at org.h2.command.Parser.parseCreate(Parser.java:5818)
	at org.h2.command.Parser.parsePrepared(Parser.java:846)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1113)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1436)
	at org.h2.server.web.WebApp.query(WebApp.java:1111)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1073)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1060)
	at org.h2.server.web.WebThread.process(WebThread.java:173)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.FileNotFoundException: Item_table_1.xlsx (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at org.h2.store.fs.FilePathDisk.newInputStream(FilePathDisk.java:337)
	at org.h2.store.fs.FileUtils.newInputStream(FileUtils.java:224)
	at org.h2.tools.Csv.initRead(Csv.java:312)
	at org.h2.tools.Csv.readResultSet(Csv.java:216)
	at org.h2.tools.Csv.read(Csv.java:192)
	... 27 more
2020-10-21 18:45:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "IOException reading Item_table_1.csv"; SQL statement:
CREATE TABLE ITEM AS SELECT * FROM CSVREAD('Item_table_1.csv') [90028-199]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:427)
	at org.h2.tools.Csv.convertException(Csv.java:586)
	at org.h2.tools.Csv.read(Csv.java:194)
	at org.h2.expression.function.Function.getValueForColumnList(Function.java:2728)
	at org.h2.table.FunctionTable.<init>(FunctionTable.java:60)
	at org.h2.command.Parser.readTableFunction(Parser.java:1945)
	at org.h2.command.Parser.readTableFilter(Parser.java:1892)
	at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:2641)
	at org.h2.command.Parser.parseSelectSimple(Parser.java:2788)
	at org.h2.command.Parser.parseSelectSub(Parser.java:2636)
	at org.h2.command.Parser.parseSelectUnion(Parser.java:2469)
	at org.h2.command.Parser.parseSelect(Parser.java:2440)
	at org.h2.command.Parser.parseCreateTable(Parser.java:7906)
	at org.h2.command.Parser.parseCreate(Parser.java:5818)
	at org.h2.command.Parser.parsePrepared(Parser.java:846)
	at org.h2.command.Parser.parse(Parser.java:788)
	at org.h2.command.Parser.parse(Parser.java:760)
	at org.h2.command.Parser.prepareCommand(Parser.java:683)
	at org.h2.engine.Session.prepareLocal(Session.java:627)
	at org.h2.engine.Session.prepareCommand(Session.java:565)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1292)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:217)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:1113)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1436)
	at org.h2.server.web.WebApp.query(WebApp.java:1111)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1073)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1060)
	at org.h2.server.web.WebThread.process(WebThread.java:173)
	at org.h2.server.web.WebThread.run(WebThread.java:93)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.FileNotFoundException: Item_table_1.csv (The system cannot find the file specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at org.h2.store.fs.FilePathDisk.newInputStream(FilePathDisk.java:337)
	at org.h2.store.fs.FileUtils.newInputStream(FileUtils.java:224)
	at org.h2.tools.Csv.initRead(Csv.java:312)
	at org.h2.tools.Csv.readResultSet(Csv.java:216)
	at org.h2.tools.Csv.read(Csv.java:192)
	... 27 more
